src/core/psxmem.cc

```diff
@@ -133,65 +140,86 @@ void PCSX::Memory::reset() {
 
     int index = 0;
     for (auto w : nobios) {
-        m_psxR[index++] = w & 0xff;
+        m_bios[index++] = w & 0xff;
         w >>= 8;
-        m_psxR[index++] = w & 0xff;
+        m_bios[index++] = w & 0xff;
         w >>= 8;
-        m_psxR[index++] = w & 0xff;
+        m_bios[index++] = w & 0xff;
         w >>= 8;
-        m_psxR[index++] = w & 0xff;
+        m_bios[index++] = w & 0xff;
         w >>= 8;
     }
-    strcpy((char *)m_psxR + index, _(R"(
+    strcpy((char *)m_bios + index, _(R"(
                    No BIOS loaded, emulation halted.
 
 Set a BIOS file into the configuration, and do a hard reset of the emulator.
 The distributed OpenBIOS.bin file can be an appropriate BIOS replacement.
 )"));
 
     // Load BIOS
-    auto &biosPath = g_emulator->settings.get<Emulator::SettingBios>().value;
-    IO<File> f(new PosixFile(biosPath.string()));
-    if (f->failed()) {
-        g_system->printf(_("Could not open BIOS:\"%s\". Retrying with the OpenBIOS\n"), biosPath.string());
-
-        g_system->findResource(
-            [&f](const std::filesystem::path &filename) {
-                f.setFile(new PosixFile(filename));
-                return !f->failed();
-            },
-            "openbios.bin", "resources", std::filesystem::path("src") / "mips" / "openbios");
+    {
+        auto &biosPath = g_emulator->settings.get<Emulator::SettingBios>().value;
+        IO<File> f(new PosixFile(biosPath.string()));
         if (f->failed()) {
-            g_system->printf(_(
-                "Could not open OpenBIOS fallback. Things won't work properly.\nAdd a valid BIOS in the configuration "
-                "and hard reset.\n"));
-        } else {
-            biosPath = f->filename();
+            g_system->printf(_("Could not open BIOS:\"%s\". Retrying with the OpenBIOS\n"), biosPath.string());
+
+            g_system->findResource(
+                [&f](const std::filesystem::path &filename) {
+                    f.setFile(new PosixFile(filename));
+                    return !f->failed();
+                },
+                "openbios.bin", "resources", std::filesystem::path("src") / "mips" / "openbios");
+            if (f->failed()) {
+                g_system->printf(
+                    _("Could not open OpenBIOS fallback. Things won't work properly.\nAdd a valid BIOS in the "
+                      "configuration "
+                      "and hard reset.\n"));
+            } else {
+                biosPath = f->filename();
+            }
+        }
+
+        if (!f->failed()) {
+            f->read(m_bios, bios_size);
+            f->close();
+            PCSX::g_system->printf(_("Loaded BIOS: %s\n"), biosPath.string());
         }
     }
 
-    if (!f->failed()) {
-        f->read(m_psxR, bios_size);
-        f->close();
-        PCSX::g_system->printf(_("Loaded BIOS: %s\n"), biosPath.string());
+    // Load EXP1
+    {
```

Comment:
Maybe let's keep a boolean somewhere to indicate the exp1 is loaded with something. If not loaded, we should nuke the corresponding LUTs.